src/modules/hip/kernel/tensor_stddev.hpp

```diff
@@ -0,0 +1,501 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+#include "reduction.hpp"
+
+// ----------------------- Helper Functions --------------------------
+
+__device__ void stddev_hip_compute(uchar *srcPtr, float src, float *dst) { *dst = src; }
+__device__ void stddev_hip_compute(float *srcPtr, float src, float *dst) { *dst = src * 255; }
+__device__ void stddev_hip_compute(signed char *srcPtr, float src, float *dst) { *dst = src; }
```

Comment:
stdDev is technically being computed outside. This seems more like a bit_depth_adjustment_hip_compute()
Also, src is passed by value, and dst by reference. It will create a variable copy in every thread. Suggest referencing previously allocated smem itself.

Reply:
I changed it, but i am not sure why this multiplication by 255 is added by snehaa for float and float16
It is not there for tensor_mean hip kernel

@r-abishek please let me know if these needs to be removed